installer_scripts/install-yb-voyager

```diff
@@ -832,6 +835,25 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	script_dir_path="guardrail_scripts"
+	scripts_dir_name="guardrail-scripts"
+	sudo mkdir -p $scripts_parent_dir/$scripts_dir_name
+
+	output "Installing the latest guardrail scripts"
+	if [ "${VERSION}" == "latest" ]
+	then
+		pg_permissions_check_script_url="https://raw.githubusercontent.com/yugabyte/yb-voyager/shivansh/guardrail-scripts-installer-script/guardrail_scripts/check_pg_permissions.sql"
+		sudo wget -nv -O $scripts_parent_dir/$scripts_dir_name/check_pg_permissions.sql $pg_permissions_check_script_url
```

Comment:
remember to change the link before merging. 
Why did you even have to add this? tests anyway install with -l, right?

Reply:
Yeah let me change it right now only